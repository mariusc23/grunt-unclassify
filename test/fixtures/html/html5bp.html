<div class="browserupgrade"></div>
<div class="hidden"></div>
<div class="visuallyhidden"></div>
<div class="focusable"></div>
<div class="invisible"></div>
<div class="clearfix"></div>
