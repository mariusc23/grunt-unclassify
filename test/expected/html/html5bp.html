<div></div>
<div class="hidden"></div>
<div></div>
<div></div>
<div></div>
<div></div>
